docs/faq.md

```diff
@@ -81,3 +81,13 @@ sudo systemctl disable cloud4rpi.service
 sudo rm /lib/systemd/system/cloud4rpi.service
 sudo systemctl daemon-reload
 ```
+
+
+## How to choose the DATA_SENDING_INTERVAL value?
+
+You should call [device.publish_data()](/api/python/#publish_data) and [device.publish_diag()](/api/python/#publish_diag) functions frequently to update the monitored variables. The update frequency depends on the variable dynamics, for example, atmospheric conditions do not change every minute. This means the delay between updates for a weather station should be one to ten minutes.
+
+If you monitor several variables with different dynamics, the delay between updates should be the shortest.
```

Comment:
I can see you do not understand. 
1. "the delay between updates" Which delay'?
2. "should be the shortest of all" 'the largest', 'the most', 'the shortest', etc. do not require 'of all', and in this case, it isn't even clear what 'of all' refers to. What are you trying to say here?